8/changelog.yaml

```diff
@@ -127,3 +127,7 @@ releases:
         from Ansible 10 if no one starts maintaining it again before Ansible 10. See
         `the removal process for details on how this works <https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection>`__
         (https://github.com/ansible-community/community-topics/issues/235).
+      - The ngine_io.vultr collection is officially unmaintained and has been archived. Therefor, it will be removed from Ansible 9.
```

Comment:
```suggestion
      - The ngine_io.vultr collection is officially unmaintained and has been archived. Therefore, it will be removed from Ansible 9.
```